src/main/resources/assets/index.htm

```diff
@@ -136,11 +136,18 @@
 				</div>	
 			</div>
 		  
+			
 		  	<div class="row">
 				<div class="col-xs-12 col-md-6 form-group">
-					<label for="ref_status">Status: </label> 
-					<input id="ref_status" type="text" class="form-control" value="Open" disabled/>
-				</div>							
+					<label for="ref_status">Status</label>
+					<select id="ref_status" class="form-control">
```

Comment:
Try to use Dropdown component from Bootstrap.
[Here](https://getbootstrap.com/docs/3.3/components/#dropdowns) you can find some example.